src/views/GameRoom/components/CardDeck/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { PlayingCard } from '../../../../components/PlayingCard';
+import { CardsWrapper } from './styles';
+
+const cards = Array(30).fill({
```

Comment:
Rendering a lot of elements with shadow will go heavy on the DOM. We should improve the PlayingCards later.